### ISSUE TITLE
Hash routes in 8.10.0

### ISSUE THREAD

**Issue by gilphilbert** [krasimir/navigo#283]
I was using 8.9.1, which worked well except I was getting double entries in my history (and navigating "back" was a little odd...) so I upgraded to 8.10.0. Now my history works well... but that's because hash routing isn't working at all!

I haven't changed anything:
```html
<ul class="menu-list has-text-centered">
    <li><a href="/" data-navigo>Playing</li>
    <li><a href="/playlists" data-navigo>Playlists</li>
    ...
</ul>
```

```javascript
self.router = new window.Navigo('/', { hash: true })
self.router
  .on('/playlists', () => {
    // do some stuff
  })
```

It seems like Navigo isn't intercepting the routes as they're acting like normal anchors and redirecting the page. If I turn off hash-based routing it works exactly as expected.

Any suggestions?

**Comment by krasimir** [comment]
This seems like a bug. I'll investigate.

**Comment by krasimir** [comment]
Can you please try the new `8.10.1`.

**Comment by gilphilbert** [comment]
Just tried the new version: hash-based routing works again... but now I'm getting three entries in my history per route. I checked with a simple `console.log` and the event is only firing once.

**Comment by krasimir** [comment]
Yep. This must be fixed. I suspect that it's like that for long time.

**Comment by krasimir** [comment]
Hey, sorry for the delay. The problem should be fixed in the latest `8.11.1` version.
